dataclients/kubernetes/clusterstate.go

```diff
@@ -42,11 +42,15 @@ func (state *clusterState) getServiceRG(namespace, name string) (*service, error
 	return s, nil
 }
 
+func (state *clusterState) GetEndpointsByService(namespace, name, protocol string, servicePort *servicePort) []string {
```

Comment:
Why not rename `getEndpointsByService`  to `GetEndpointsByService`?

Reply:
yes true